deps/build.jl

```diff
@@ -14,7 +14,17 @@ if is_windows()
     end
 end
 
-libhttp_parser = library_dependency("libhttp_parser", aliases=aliases)
+function validate_httpparser(name,handle)
+    try
+        p = Libdl.dlsym(handle, :http_parser_url_init)
```

Comment:
It was introduced first in 2.6.0, which was released a year and a half after the first Windows binaries were compiled.